docs/reference/commandline/build.md

```diff
@@ -29,7 +29,7 @@ parent = "smn_cli"
       -m, --memory=""                 Memory limit for all build containers
       --memory-swap=""                A positive integer equal to memory plus swap. Specify -1 to enable unlimited swap.
       --no-cache                      Do not use cache when building the image
-      --pull                          Always attempt to pull a newer version of the image
+      --pull=<content-trust>          Always attempt to pull a newer version of the image
```

Comment:
@tiborvass Yeah, I wasn't sure how to express that in the docs/help output.

`<content-trust>` means --pull will default to whatever calculated value `isTrusted()` resolves to.  So, if you didn't specify any --pull value, --pull flag would be set to `isTrusted()`.